python/ola/PidStore.py

```diff
@@ -981,14 +1044,18 @@ def LoadFile(self, pid_file_name, validate):
           manufacturer.manufacturer_name)
 
       for pid_pb in manufacturer.pid:
+        if override:  # if processing overrides delete any conflicting entry
+          old = pid_dict.pop(pid_pb.value, None)
+          if old is not None:
+            del name_dict[old.name]
```

Comment:
Presumably you're not just doing pid_pb.value in pid_dict because you need to handle the name_dict deletion?

Reply:
There's no guarantee that entities in the overrides are actually already defined, so this handles deleting if there cleanly.

Reply:
Very good point!

Reply:
I just need to check what happens in C++ if we merge an override with set only onto a normal PID with get only, do we get both or set only, as the Python should match this.

Reply:
So I did check and in C++ it override the entire PID if it exists, so any pre-existing functionality is lost and hence this is the correct behaviour.